GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -808,6 +811,46 @@ void ExporterHelper::ExportObjectAndBehaviorsIncludes(
   }
 }
 
+void ExporterHelper::ExportObjectAndBehaviorsRequiredFiles(
+    const gd::Project &project, std::vector<gd::String> &requiredFiles) {
+  auto addRequiredFiles = [&](const std::vector<gd::String> &newRequiredFiles) {
+    for (const auto &requiredFile : newRequiredFiles) {
+      
+      std::cout << "include as resource: " << requiredFile
+                << std::endl;
+      InsertUnique(requiredFiles, requiredFile);
+    }
+  };
+
+  auto addObjectRequiredFiles = [&](const gd::Object &object) {
+    // Ensure needed files are included for the object type and its behaviors.
+
+    // TODO Do the same for objects once we have a case to test it.
```

Comment:
This is to avoid to add dead code

Reply:
I don't think I understand the comment. Since the name of the method is `addObject...` I would tend to believe that we are doing it for the objects

Reply:
Let's go for:

`TODO: Handle required files declared by objects. For now, no objects has a need for additional required files, so the object metadata do not even have `requiredFiles`.`